libkbfs/md_journal.go

```diff
@@ -464,7 +464,10 @@ func (j *mdJournal) convertToBranch(
 		brmd.SetUnmerged()
 		brmd.SetBranchID(bid)
 
-		// Delete the old "merged" version from the cache.
+		// Delete the old "merged" version from the cache.  We aren't
+		// equipped here to cache the new version, but it will
+		// eventually get passed via an mdFlushListener to somewhere
+		// that will cache it.
```

Comment:
My thought is that, if it's not already in the cache, it must have been evicted for something more recently used, and it doesn't seem good to kick whatever that is out just because a branch conversion happened.